connector_prestashop/data/cron.xml

```diff
@@ -0,0 +1,95 @@
+
+    <record forcecreate="True" id="ir_cron_import_customers" model="ir.cron">
+        <field name="name">PrestaShop - Import Customers and Groups</field>
+        <field name="active" eval="False" />
+        <field name="user_id" ref="base.user_root" />
+        <field name="state">code</field>
+        <field name="code">model._scheduler_import_customers()</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="connector_prestashop.model_prestashop_backend" />
+    </record>
+
+    <record forcecreate="True" id="ir_cron_import_products" model="ir.cron">
+        <field name="name">PrestaShop - Import Products and Categories</field>
+        <field name="active" eval="False" />
+        <field name="user_id" ref="base.user_root" />
+        <field name="state">code</field>
+        <field name="code">model._scheduler_import_products()</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="connector_prestashop.model_prestashop_backend" />
+    </record>
+
+    <record forcecreate="True" id="ir_cron_import_suppliers" model="ir.cron">
+        <field name="name">PrestaShop - Import suppliers</field>
+        <field name="active" eval="False" />
+        <field name="user_id" ref="base.user_root" />
+        <field name="state">code</field>
+        <field name="code">model._scheduler_import_suppliers()</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="connector_prestashop.model_prestashop_backend" />
+    </record>
+
+    <record forcecreate="True" id="ir_cron_import_carriers" model="ir.cron">
+        <field name="name">PrestaShop - Import Carriers</field>
+        <field name="active" eval="False" />
+        <field name="user_id" ref="base.user_root" />
+        <field name="state">code</field>
+        <field name="code">model._scheduler_import_carriers()</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="connector_prestashop.model_prestashop_backend" />
+    </record>
+
+    <record forcecreate="True" id="ir_cron_import_payment_modes" model="ir.cron">
+        <field name="name">PrestaShop - Payment modes</field>
+        <field name="active" eval="False" />
+        <field name="user_id" ref="base.user_root" />
+        <field name="state">code</field>
+        <field name="code">model._scheduler_import_payment_modes()</field>
```

Comment:
This method has a wrong name.

Now is this:

#https://github.com/OCA/connector-prestashop/pull/173/files#diff-fb4b550b10a5f122d8f99cfa90a2d991a1688b1eb1bfb20a620f2c3f1eef4604R378

Suggested change:

`<field name="code">model._scheduler_import_payment_methods()</field>`